velox/core/PlanNode.cpp

```diff
@@ -159,11 +159,6 @@ AggregationNode::AggregationNode(
         "GroupId key {} must have global grouping sets",
         groupId_.value()->name());
   }
-
-  if (!globalGroupingSets_.empty()) {
```

Comment:
Let's keep this check, but change the application (Prestissimo).

Reply:
@mbasmanova : The application code ends up being quite weird. But its possible to do that if we want to keep the Velox API constrained.

Reply:
It would be nice to keep the API minimal and not allow specifying things that are to-be-ignored.

Reply:
@mbasmanova : I eventually went with the changes as per https://github.com/prestodb/presto/pull/21288 to tighten given the intent of this code. But the co-ordinator doesn't quite honor it that way.